service/dap/server.go

```diff
@@ -546,9 +582,9 @@ func (s *Server) onSetBreakpointsRequest(request *dap.SetBreakpointsRequest) {
 	i := 0
 	for _, b := range request.Arguments.Breakpoints {
 		bp, err := s.debugger.CreateBreakpoint(
-			&api.Breakpoint{File: request.Arguments.Source.Path, Line: b.Line})
+			&api.Breakpoint{File: request.Arguments.Source.Path, Line: b.Line, Cond: b.Condition})
 		if err != nil {
-			s.log.Error("ERROR:", err)
+			s.log.Error("ERROR: ", err)
```

Comment:
I took a look at the SetBreakpointsRequest specification and it seems to me that when there is an error the adapter can return `verified == false` and set the `message` field to explain why the breakpoint couldn't be verified, wouldn't that be better?

I'm approving this, this is just something to consider.

Reply:
Done